app/assets/javascripts/components.js

```diff
@@ -1 +1,3 @@
+window.$ = require('jquery')
+window.jQuery = $
```

Comment:
__[eslint]__
* Error - "window" is not defined. (no-undef)
* Error - "$" is not defined. (no-undef)